client/plots/plot.disco.js

```diff
@@ -73,7 +73,12 @@ export default async function (termdbConfig, dslabel, sample, holder, genomeObj)
 					{
 						chartType: 'Disco',
 						subfolder: 'disco',
-						extension: 'ts'
+						extension: 'ts',
+						overrides: {
+							label: {
+								showPrioritizeCancerGenes: !!genomeObj.geneset
```

Comment:
works now.
cancer gene census is just one gene set to prioritize the labels, there are many other choices. thus it's not good to call the attribute `CancerGenes` but should be a generic label e.g. `prioritizeGeneLabelsByGeneSets`
same for the folder and script names

Reply:
I've changed it and pushed.